build.gradle

```diff
@@ -36,12 +42,46 @@ allprojects {
     repositories {
         google()
         mavenCentral()
-        jcenter()
         mavenLocal()
+        jcenter() {
+            content {
+
+                // Remove when https://github.com/google/ExoPlayer/issues/5246 is closed
+                includeGroup("com.google.android.exoplayer")
+
+                // Remove when https://github.com/facebook/flipper/issues/1775 is closed
+                includeGroup("com.facebook.flipper")
+
+                // Remove when https://github.com/Kotlin/dokka/issues/41 is closed
+                includeGroup("org.jetbrains.dokka")
+
+                includeModule("net.yslibrary.keyboardvisibilityevent", "keyboardvisibilityevent")
+                includeModule("me.relex", "photodraweeview")
+                includeModule("org.jetbrains.trove4j", "trove4j")
+
+                // Remove when https://github.com/korlibs/korte/issues/13 is closed
+                includeModule("com.soywiz.korlibs.korte", "korte-jvm")
+
+                // Remove when https://github.com/Kotlin/kotlinx.html/issues/81 is closed
+                includeModule("org.jetbrains.kotlinx", "kotlinx-html-jvm")
+
+                //Remove when https://github.com/JetBrains/markdown/issues/64 is closed
+                includeModule("org.jetbrains", "markdown")
+
+                // Remove when https://github.com/MarkusAmshove/Kluent/issues/190 is closed
+                includeGroup("org.amshove.kluent")
+
+                includeGroup("androidx.databinding")
```

Comment:
Isn't this in `google()`? https://maven.google.com/web/index.html?q=databinding

Reply:
Yep, it is, I don't know why I had some issues yesterday with this dependency